pkg/cloudprovider/gcp/block_store.go

```diff
@@ -142,10 +159,11 @@ func (b *blockStore) CreateVolumeFromSnapshot(snapshotID, volumeType, volumeAZ s
 	}
 
 	if isMultiZone(volumeAZ) {
-		volumeRegion, err := parseRegion(volumeAZ)
+		volumeRegion, err := parseZone(volumeAZ)
 		if err != nil {
 			return "", err
 		}
+		disk.ReplicaZones = createZoneURLs(b.project, volumeAZ)
```

Comment:
I think there may be an option to not hardcode the URL template -- you should be able to do something like:

```
zone, err := b.gce.Zones.Get("<project>", "zone>").Do()
// TODO handle error
url := zone.SelfLink
```

The `url` value looks like it will be a relative one, e.g. "/projects/<proj>/..." -- but I *think* based on other API documentation that this should be an acceptable value for `ReplicaZones`.  If this works, I think it's probably preferable.

Reply:
Cool, thanks. I think I'd prefer that to a templated URL, too. Will give it a test.

Reply:
Not 100% sure that the relative URL will be accepted, but 🤞

Reply:
It works!
 
```
% k get pv -o yaml
apiVersion: v1
items:
- apiVersion: v1
  kind: PersistentVolume
    labels:
      failure-domain.beta.kubernetes.io/region: us-west1
      failure-domain.beta.kubernetes.io/zone: us-west1-b__us-west1-c
      velero-restore: nginx-regional-20190211133203
      velero.io/backup-name: nginx-regional
      velero.io/restore-name: nginx-regional-20190211133203
```

I'll look around how to make the unit testing work with this client call.

Reply:
Changed, though I didn't mock out the client seeing as none of the rest of that logic is mocked currently.

Reply:
Yeah, I think it's OK for now given precedent. Is this otherwise good to go? I'll do a quick test on it as well - got any sample YAMLs I could use?